### PR TITLE
Refac build configs

### PR DESCRIPTION
**Pull request by Ashuaidehao**
Close https://github.com/neo-project/neo-modules/issues/848

### CODE REVIEW

**Comment by Liaojinghui** [comment]
Why is this pr?

**Comment by superboyiii** [comment]
See https://github.com/neo-project/neo-modules/issues/848 @shargon @Liaojinghui

**Review by Liaojinghui — APPROVED** [review]

**Comment by Liaojinghui** [comment]
Behaves the same as description #848

**Review by vncoelho — COMMENTED** [review]
Looks good change!

**Comment by vncoelho** [comment]
I will compile here and see results.

**Review by Liaojinghui — DISMISSED** [review]

**Comment by superboyiii** [comment]
Choose #849 or this? They have the same behaviour.

**Comment by shargon** [comment]
This was first, isn't it?

**Comment by Liaojinghui** [comment]
> This was first, isn't it?

Yes

**Comment by cschuchardt88** [comment]
however when you `dotnet build` it will build all the files. and i thought you wanted no dependencies in the output. #849 doesn't have that problem in workflow actions.

File: src/Directory.Build.props [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Neo" Version="3.6.2" />
-  </ItemGroup>
-
-</Project>
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+	<ItemGroup>
+		<None Include="config.json" Condition="Exists('config.json') And '$(IncludeSettingsFileOutput)' != 'False'">
```
Comment (AnnaShaleva) [inline src/Directory.Build.props:5]: We've excluded `IncludeSettingsFileOutput` from every project, so I think we don't need this condition (`'$(IncludeSettingsFileOutput)' != 'False'`) to be presented here, it's just not relevant anymore.

Is `config.json` a configuration file of the module's dependent library? If so, then we probably just don't need to copy it while publishing module.

**Review by AnnaShaleva — COMMENTED** [review]
Nice enhancement, we'll port it to neo-bench.

File: src/Directory.Build.props (hunk 1 above)
Comment (Liaojinghui) [reply src/Directory.Build.props:5]: > We've excluded `IncludeSettingsFileOutput` from every project

Not every project, just part of them.

Comment (AnnaShaleva) [reply src/Directory.Build.props]: I mean from every neo-modules project. We don't have any mentions of `IncludeSettingsFileOutput=False` anymore in neo-modules.

**Review by AnnaShaleva — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Comment by Liaojinghui** [comment]
@Ashuaidehao this breaks the publish process, please check and fix.

**Pull request merged**